<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GPS</class>
 <widget class="QWidget" name="GPS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Format:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="format_combobox">
       <item>
        <property name="text">
         <string>DD° MM’ SS.sss’’</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DD.dddddd°</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DD° MM.mmmmm’</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Latitude:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Longitude:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="dop_label">
           <property name="text">
            <string>VDOP / HDOP / PDOP:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Estimated position error:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="latitude">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Unknown</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ns">
             <property name="text">
              <string>U</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="longitude">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Unknown</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ew">
             <property name="text">
              <string>U</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="dop">
           <property name="text">
            <string>Unknown</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="epe">
           <property name="text">
            <string>Unknown</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="show_pos">
       <property name="text">
        <string>Show Position in Google Maps</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Fix:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Satellites in view:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Satellites used:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Altitude:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Course:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Speed:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Time:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Battery Voltage:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="fix">
         <property name="text">
          <string>No</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="satellites_view">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="satellites_used">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="altitude">
         <property name="text">
          <string>Unknown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="course">
         <property name="text">
          <string>Unknown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="speed">
         <property name="text">
          <string>Unknown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="time">
         <property name="text">
          <string>Unknown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="voltage">
         <property name="text">
          <string>0 V</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="hot_start">
       <property name="text">
        <string>Hot Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="warm_start">
       <property name="text">
        <string>Warm Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cold_start">
       <property name="text">
        <string>Cold Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="factory_reset">
       <property name="text">
        <string>Factory Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
